README.md

```diff
@@ -69,24 +69,28 @@ These properties should be treated with care, as in general changing them will c
 
 The return value of "failure" in the spec is represented by `null`. That is, functions like `parseURL` and `basicURLParse` can return _either_ a URL record _or_ `null`.
 
+### `whatwg-url/webidl2js-wrapper` module
+
+This module exports the `URL` and `URLSearchParams` [interface wrappers API](https://github.com/jsdom/webidl2js#for-interfaces) generated by [webidl2js](https://github.com/jsdom/webidl2js).
+
 ## Development instructions
 
 First, install [Node.js](https://nodejs.org/). Then, fetch the dependencies of whatwg-url, by running from this directory:
 
-    npm install
+    yarn
```

Comment:
We need to update the sentence above to talk about installing Yarn too.

And with that, I worry a bit about this change, because these instructions were originally added in https://github.com/jsdom/whatwg-url/pull/118 to make it clearer for folks involved in the URL spec ecosystem, but not the Node.js ecosystem. Telling them they need to get two programs instead of one doubles the potential burden, for somewhat minimal gain.

Maybe it's fine. I do appreciate the consistency across the jsdom ecosystem.

An alternative would be to leave these instructions but still switch from package-lock.json to yarn.lock. The theory being that the maintainers who are doing stuff like upgrading deps can figure it out anyway, and ignore the README instructions, whereas people doing URL-spec-related hacking can use the README instructions.

Reply:
If that is the case, I'd rather have us remain in npm sphere.

> An alternative would be to leave these instructions but still switch from package-lock.json to yarn.lock.

It's confusing to have two lock files in the same directory though. Yarn (though not npm) complains if the opposite lockfile exists:

> **warning** package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.